roles/network_plugin/multus/tasks/main.yml

```diff
@@ -11,6 +11,14 @@
     - {name: multus-clusterrolebinding, file: multus-clusterrolebinding.yml, type: clusterrolebinding}
   register: multus_manifest_1
 
+# Calico supports CNI more then 0.3.1 only after v3.23.0 https://github.com/projectcalico/calico/pull/5466
+- name: check for calico version
+  set_fact:
+    multus_cni_version: "0.3.1"
```

Comment:
If we can say "0.3.2 never come in the future", it is fine to go with current code.
My concern is if releasing 0.3.2 or more and specifying the newer version on the inventory file, this code overwrites this older version.
That would cause confusions.

But if such situation never happens, current code is fine.

Reply:
It's doubtful this would come before we move beyond calico 3.23.x as which point we will probably drop this hack anyway.